aten/src/ATen/core/type.cpp

```diff
@@ -468,23 +468,6 @@ bool Type::isSubtypeOf(const TypePtr rhs) const {
   return *this == *rhs;
 }
 
-ClassTypePtr ClassType::create(
```

Comment:
I don't see the need for this change it is going to conflict with https://github.com/pytorch/pytorch/pull/21814. Can it be avoided?

Reply:
These need to be moved out of c10 because otherwise they cause undefined reference to vtable of ClassType on android 🙄